chalicelib/checks/wfr_checks.py

```diff
@@ -754,59 +1050,81 @@ def SNV_metawfrs_to_patch_sample_processing(connection, **kwargs):
                     SV_processed += 1
                 elif pf['variant_type'] == "SNV":
                     SNV_processed += 1
-            except:
+            except Exception: # SNV probably should be specified and not just be the default
                 SNV_processed += 1
-        if SNV_processed < 2:
+        if SNV_processed < 2: # 2 because there are 2 processed_files when it's patched already
+                              #     and so we want to skip that case
             filtered_res.append(r)
 
-    # nothing to run
+    # if nothing to run, return
     if not filtered_res:
         check.summary = 'All Good!'
         return check
 
+    # else
     metawfr_uuids = [r['meta_workflow_run']['uuid'] for r in filtered_res]
     metawfr_titles = [r['meta_workflow_run']['display_title'] for r in filtered_res]
 
+    ### More check attributes, setting up the action
     check.allow_action = True
     check.summary = 'Some metawfrs may need patching sample processing.'
     check.status = 'WARN'
     msg = str(len(metawfr_uuids)) + ' metawfrs may need patching sample processing'
     check.brief_output.append(msg)
     check.full_output['metawfrs_to_check'] = {'titles': metawfr_titles, 'uuids': metawfr_uuids}
-    return check
 
+    return check
 
 @action_function()
 def patch_SNV_pfs_to_sample_processing(connection, **kwargs):
+    ### General action attributes
     start = datetime.utcnow()
     action = ActionResult(connection, 'patch_SNV_pfs_to_sample_processing')
-    action_logs = {'runs_checked_for_patching': []}
+    action_logs = {'runs_checked_for_patching': [], 'runs_checked_for_patching_log': []}
     my_auth = connection.ff_keys
     env = connection.ff_env
     check_result = action.get_associated_check_result(kwargs).get('full_output', {})
     action_logs['check_output'] = check_result
+    action_logs['error'] = []
     metawfr_uuids = check_result.get('metawfrs_to_check', {}).get('uuids', [])
     random.shuffle(metawfr_uuids)  # if always the same order, we may never get to the later ones.
+
+    ### Running the action
     for metawfr_uuid in metawfr_uuids:
         now = datetime.utcnow()
         if (now-start).seconds > lambda_limit:
             action.description = 'Did not complete action due to time limitations'
             break
         try:
-            patch_SNV_processed_files_to_sample_processing(metawfr_uuid, my_auth)
+            # action function call
+            res = patch_processed_files_to_sample_processing(metawfr_uuid,
+                ['workflow_vep-annot-check', 'workflow_hg19lo_hgvsg-check'],
+                my_auth)
             action_logs['runs_checked_for_patching'].append(metawfr_uuid)
+            action_logs['runs_checked_for_patching_log'].append(metawfr_uuid + ' ' + res)
         except Exception as e:
-            action_logs['error'] = str(e)
-            break
+            action_logs['error'].append(str(e))
+            continue
```

Comment:
Are we positive it's a good idea to `continue` here? Might make more sense to stop immediately, but not positive.